<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>STOK PROGRAMI</string>
    </property>
    <widget class="QPushButton" name="pushButton_giris">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Malzeme Girişi</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cikis">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Malzeme Çıkışları</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stok">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Stok Durumunu Göster</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuproje_y_kle">
    <property name="title">
     <string>Proje yükle</string>
    </property>
    <addaction name="actionYeni_proje"/>
    <addaction name="actionProje_yukle"/>
    <addaction name="actionSon_pencere"/>
    <addaction name="separator"/>
    <addaction name="actionProjeyi_kaydet"/>
    <addaction name="action_cik"/>
   </widget>
   <widget class="QMenu" name="menu_lemler">
    <property name="title">
     <string>İşlemler</string>
    </property>
    <addaction name="actionMalzeme_giris"/>
    <addaction name="actionMalzeme_cikis"/>
    <addaction name="actionStok_durumu"/>
    <addaction name="actionTutanak_olustur"/>
   </widget>
   <widget class="QMenu" name="menuYard_m">
    <property name="title">
     <string>Yardım</string>
    </property>
    <addaction name="actionHakk_nda"/>
    <addaction name="actionG_ncelleme"/>
    <addaction name="actionDestek_Al"/>
   </widget>
   <addaction name="menuproje_y_kle"/>
   <addaction name="menu_lemler"/>
   <addaction name="menuYard_m"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionYeni_proje">
   <property name="text">
    <string>Yeni proje oluştur</string>
   </property>
  </action>
  <action name="actionProje_yukle">
   <property name="text">
    <string>Proje yükle</string>
   </property>
  </action>
  <action name="actionSon_pencere">
   <property name="text">
    <string>Son pencereyi Aç</string>
   </property>
  </action>
  <action name="actionProjeyi_kaydet">
   <property name="text">
    <string>Projeyi Kaydet</string>
   </property>
  </action>
  <action name="action_cik">
   <property name="text">
    <string>Çıkış</string>
   </property>
  </action>
  <action name="actionMalzeme_giris">
   <property name="text">
    <string>Malzeme Girişi</string>
   </property>
  </action>
  <action name="actionMalzeme_cikis">
   <property name="text">
    <string>Malzeme Çıkışı</string>
   </property>
  </action>
  <action name="actionStok_durumu">
   <property name="text">
    <string>Stok Durumu</string>
   </property>
  </action>
  <action name="actionTutanak_olustur">
   <property name="text">
    <string>Tutanak Oluştur</string>
   </property>
  </action>
  <action name="actionHakk_nda">
   <property name="text">
    <string>Hakkında</string>
   </property>
  </action>
  <action name="actionG_ncelleme">
   <property name="text">
    <string>Güncelleme</string>
   </property>
  </action>
  <action name="actionDestek_Al">
   <property name="text">
    <string>Destek Al</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
